deltachat-ios/Controller/Settings/SelfProfileViewController.swift

```diff
@@ -47,10 +56,16 @@ class SelfProfileViewController: UITableViewController, MediaPickerDelegate {
     }()
 
     init(dcAccounts: DcAccounts) {
-        self.dcAccounts = dcAccounts
         self.dcContext = dcAccounts.getSelected()
         super.init(style: .insetGrouped)
         hidesBottomBarWhenPushed = true
+
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(SelfProfileViewController.textDidChangeNotification(notification:)),
```

Comment:
I don't mind it but ftr you don't have to be this verbose. This is also possible I think.
```suggestion
            selector: #selector(textDidChangeNotification),
```

Reply:
cool! did not knew that (just did copy+paste :)